static/lang/re-en.json

```diff
@@ -3680,6 +3680,23 @@
                     "Success": "If the target is evil, it is also @UUID[Compendium.pf2e.conditionitems.Item.MIRkyAjyBeXivMa7]{Enfeebled 1} until the start of your next turn on a hit."
                 }
             },
+            "ResonantReflection": {
+                "ReflectionOfLife": {
+                    "Description": "**Effect** You gain fast healing 3 for 1 minute. This amount of fast healing increases by 1 for every 2 levels you have beyond 5th level</p>\n<p>@UUID[Compendium.pf2e.campaign-effects.Item.DOxl0FDd21VMDOMP]{Effect: Reflection of Life (Fast Healing)}."
+                },
+                "ReflectionOfLight": {
+                    "Description": "**Effect** You cast @UUID[Compendium.pf2e.spells-srd.Item.DyiD239dNS7RIxZE]{Holy Light} as an innate divine spell, heightened to a spell rank equal to half your level rounded up."
+                },
+                "ReflectionOfStone": {
+                    "Description": "**Effect** You cast @UUID[Compendium.pf2e.spells-srd.Item.2BV2yYPfVJ5zirZt]{Mountain Resilience} as an innate divine spell, heightened to a spell rank equal to half your level rounded up."
+                },
+                "ReflectionOfStorm": {
+                    "Description": "**Effect** You cast @UUID[Compendium.pf2e.spells-srd.Item.4MOew29Z1oCX8O28]{Moment of Renewal} (self only) as an 8th-rank divine innate spell and regain 1 Focus Point."
+                },
+                "ReflectionOfWater" : {
+                    "Description": "**Effect** You cast @UUID[Compendium.pf2e.spells-srd.Item.zfn5RqAdF63neqpP]{Control Water} as a 5th-rank innate divine spell. The area of water you control increases by 10 feet in length and width for every 2 levels you have beyond 10th level."
+                }
+            },
```

Comment:
Add title fields for each of them and remove the `**Effect**`.

Reply:
Going with the suggestion.